zombies.js

```diff
@@ -270,7 +420,12 @@
  * -----------------------------
  */
 
-
+class StrongZombie extends Zombie{
+  constructor(health, strength, speed){
+    super(health, strength, speed);
+    this.speed = speed;
```

Comment:
speed is part of the super constructor, so I does not need to be set on the child